index.html

```diff
@@ -0,0 +1,99 @@
+<!DOCTYPE html>
+<html lang="en">
+<head>
+    <meta charset="UTF-8">
+    <meta name="viewport" content="width=device-width, initial-scale=1.0">
+    <link rel="icon" href="img/icons8-blockchain.png" type="image/x-icon">
+    <link rel="preconnect" href="https://fonts.googleapis.com">
+    <link rel="preconnect" href="https://fonts.gstatic.com" crossorigin>
+    <link href="https://fonts.googleapis.com/css2?family=Lato:ital,wght@0,300;0,400;0,700;1,100;1,300&display=swap"
+          rel="stylesheet">
+    <link rel="stylesheet " href="css/style.css" >
+    <title>Crypto Con &mdash; Your Blockchain Conference</title>
+</head>
+<body>
+<div class="hero">
```

Comment:
- [ ] **Required:** This sections function appropriately. However, you are required to use more semantic tags throughout the website. Kindly replace most or all of the div tags in the project with more semantic tags.

![image](https://user-images.githubusercontent.com/27709832/162815012-b1a03e7e-5c4f-4f4b-8edb-08d2b921af6d.png)